modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -39,30 +39,22 @@ const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
 			onInstanceReady={(event) => {
 				const editor = event.editor;
 
-				const balloonToolbars = editor.balloonToolbars;
+				const toolbars = config.toolbars;
 
-				balloonToolbars.create({
-					buttons:
-						'Bold,Italic,Underline,RemoveFormat,Link,NumberedList,BulletedList,JustifyLeft,JustifyCenter,JustifyRight,JustifyBlock,Anchor',
-					cssSelector: '*',
-				});
+				if (toolbars) {
+					const balloonToolbars = editor.balloonToolbars;
 
-				balloonToolbars.create({
-					buttons: 'Link,Unlink',
-					priority:
-						window.CKEDITOR.plugins.balloontoolbar.PRIORITY.HIGH,
-					refresh(editor, path) {
-						return path.contains('a');
-					},
-				});
-
-				balloonToolbars.create({
-					buttons:
-						'JustifyLeft,JustifyCenter,JustifyRight,Link,Unlink',
-					priority:
-						window.CKEDITOR.plugins.balloontoolbar.PRIORITY.HIGH,
-					widgets: 'image,image2',
-				});
+					toolbars.forEach((toolbar) => {
+						if (toolbar.priority) {
+							toolbar = {
+								...toolbar, 
+								priority:
+									window.CKEDITOR.plugins.balloontoolbar.PRIORITY.HIGH
```

Comment:
Do you think we could also define the same "priority" constants in Java like they're defined [here](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_plugins_balloontoolbar.html#property-PRIORITY), for this to be a bit more flexible?

Reply:
As this is part of the [Context Definition](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_plugins_balloontoolbar_contextDefinition.html#property-priority) for the balloon toolbars, as well as the `cssSelector` and the `widget` properties, I don't think this should be configurable, but must be hardconded in our js component. In addition, changing the buttons of the toolbar will not affect to its prioriry (or shouldn't). 

So, for me, only toolbar buttons should be configurable by the users.

Reply:
Thanks for the feedback @ambrinchaudhary. What you wrote also makes sense to me, so let's keep this hardcoded for now, and if we ever think there's a need for customization, we can do that later.